deploy/html/viur_base.html

```diff
@@ -87,7 +87,7 @@
 	<article class="wrap">
 		<div class="bind flex">
 
-			<article class="content {{pageContent}}" role="main"> {# name this content class after the page name AND function for better css addressing, e.g.: 'index', 'page_view view', 'product_list list' #}
+			<div class="content {{pageContent}}" role="main"> {# name this content class after the page name AND function for better css addressing, e.g.: 'index', 'page_view view', 'product_list list' #}
```

Comment:
Instead of `<div role="main">` we should use `<main>` here.

Reply:
Agreed, I have done the change.